auto_nag/scripts/reminder.py

```diff
@@ -0,0 +1,146 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+
+from dateutil import parser
+from libmozdata.bugzilla import BugzillaUser
+from libmozdata import utils as lmdutils
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+REMINDER_TYPES = [
+    "test",
+    "pref",
+]
+DATE_MATCH = re.compile(
+    r"\[reminder\-(%s) ([0-9]{4}-[0-9]{1,2}-[0-9]{1,2})\]" % "|".join(REMINDER_TYPES)
+)
+
+
+class Reminder(BzCleaner):
+    def __init__(self):
+        super(Reminder, self).__init__()
+        self.extra_ni = {}
+        self.autofix_whiteboard = {}
+
+    def description(self):
+        return "Bugs with whiteboard reminders"
+
+    def get_bz_params(self, date):
+        self.today = lmdutils.get_date_ymd(date)
+
+        params = {
+            "include_fields": ["assigned_to", "whiteboard", "triage_owner", "history"],
+            "f1": "status_whiteboard",
+            "o1": "substring",
+            "v1": "[reminder-",
+        }
+
+        return params
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+
+        new_whiteboard = whiteboard = bug["whiteboard"]
+        matches = DATE_MATCH.findall(whiteboard)
+
+        # We support having multiple reminders that expire on a single day.
+        # (I'm not sure why, but I guess we should.)
+        reminders = []
+
+        for m in matches:
+            tag, date = m
+
+            target = parser.parse(date)
+
+            if target.date() < self.today.date():
```

Comment:
```suggestion
            if lmdutils.get_date_ymd(date) < self.today:
```
Nit: for consistency